image/manifest_test.go

```diff
@@ -112,7 +112,7 @@ func TestUnpackLayer(t *testing.T) {
 			Digest:    fmt.Sprintf("sha256:%s", fmt.Sprintf("%x", h.Sum(nil))),
 		}},
 	}
-	err = testManifest.unpack(newPathWalker(tmp1), filepath.Join(tmp1, "rootfs"))
+	err = testManifest.unpack(newPathWalker(tmp1), filepath.Join(tmp1, "rootfs"), false)
```

Comment:
Can we get a new test for `preserveOwnership == true`?  I'm not sure what Go's test framework includes for skipping tests, but it would be nice to run that if the tests were run as root.